gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
+#Tue Apr 16 07:28:06 CEST 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
```

Comment:
Is the gradle update necessary? I would prefer to keep such changes separate in case they need to be reverted